cli/src/commands/extensions/api.rs

```diff
@@ -239,17 +240,18 @@ async fn get_refresh_token(op_state: Rc<RefCell<OpState>>) -> Result<RefreshToke
 }
 
 /// Retrieve a job's status.
+///
 /// Equivalent to `phylum history job`.
 #[op]
 async fn get_job_status(
     op_state: Rc<RefCell<OpState>>,
     job_id: String,
-) -> Result<JobStatusResponse<PackageStatusExtended>> {
+) -> Result<PolicyEvaluationResponse> {
     let state = ExtensionState::from(op_state);
     let api = state.api().await?;
 
     let job_id = JobId::from_str(&job_id)?;
-    api.get_job_status_ext(&job_id).await.map_err(Error::from)
+    api.get_job_status(&job_id, []).await.map_err(Error::from)
```

Comment:
We should probably either add a new API, or add an optional argument here, but I think we can postpone that / track it in an issue considering that currently no extensions should require it just yet.